src/Libraries/RevitNodes/GeometryConversion/RevitToProtoSolid.cs

```diff
@@ -28,10 +28,22 @@ public static class RevitToProtoSolid
             {
                 srfs.AddRange(face.ToProtoType(false));
             }
-            var converted = Solid.ByJoinedSurfaces(srfs);
+
+            Solid converted = null;
+            try
+            {
+                converted = Solid.ByJoinedSurfaces(srfs);
+            }
+            catch (Exception e)
```

Comment:
@Randy-Ma as discussed could you try to catch `ApplicationException` and log the specific geometry error message and in `catch(Exception e) { }` block, log a localized message saying `Solid.ByJoinedSurfaces failed` or something to that extent.